design/iterations.go

```diff
@@ -85,6 +88,24 @@ var _ = a.Resource("iteration", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+	a.Action("update", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.PATCH("/:id"),
```

Comment:
thanks @aslakknutsen @kwk for inspection, changed in https://github.com/almighty/almighty-core/pull/652/commits/8a1a3a40c901318fb3a485329f86eff0034cd4dd. Also in future APIs will use precise named parameter.